server/src/main/java/org/elasticsearch/transport/RemoteClusterService.java

```diff
@@ -309,13 +309,13 @@ protected void updateRemoteCluster(String clusterAlias, Settings settings) {
         CountDownLatch latch = new CountDownLatch(1);
         updateRemoteCluster(clusterAlias, settings, ActionListener.runAfter(new ActionListener<>() {
             @Override
-            public void onResponse(Void o) {
-                logger.debug("connected to new remote cluster [{}]", clusterAlias);
+            public void onResponse(RemoteClusterConnectionStatus status) {
+                logger.info("remote cluster connection [{}] updated: {}", clusterAlias, status);
```

Comment:
I wonder if this could be too noisy in certain scenarios?

Reply:
To clarify this:
* it is only logged when updating the relevant setting, or other cluster settings are not triggering this message
* It is not logged when starting or stopping the node
* It is logged from EVERY node of the cluster. I think it is better to keep it this way rather then make it look like only elected master is trying to connect to remote.